nrfx/drivers/src/nrfx_gpiote.c

```diff
@@ -53,6 +60,14 @@
 #define MAX_PIN_NUMBER NRFX_FOREACH_PRESENT(P, GPIO_PIN_NUM, (+), (0), _)
 #endif
```

Comment:
It looks like the conflict in this file was not resolved properly. There should be no `#endif` here.

Reply:
fixed